### ISSUE TITLE
SpeechUT does not have a link for download

### ISSUE THREAD

**Issue by world1tree** [microsoft/SpeechT5#81]
It seems that the relevant download link for SpeechUT was removed after a certain commit(47e10198f18ed4e4e7a2053bc064c860f5bda08a). Is there currently a way to download the SpeechUT model？

**Comment by world1tree** [comment]
@wszlong

**Comment by world1tree** [comment]
Is there anyone who can help me? thanks!

**Comment by lysanderism** [comment]
do you slove this problem ? and get SpeechUT model?

**Comment by world1tree** [comment]
> do you slove this problem ? and get SpeechUT model?

No! I use [WavLM](https://github.com/microsoft/unilm/tree/master/wavlm), hope it helps you.
